src/main/resources/db/migration/V1_53__referral_selected_desired_outcomes.sql

```diff
@@ -0,0 +1,14 @@
+begin;
```

Comment:
just noticed this lonely `begin`, what does it do?

Reply:
i guess it goes with the `commit` at the end?

Reply:
It wraps up the whole set of statements into a single transaction so rolls back if for instance the update fails.

It was more for when I was running directly on the database for testing - here it's not really needed but shouldn't hurt to have it

Reply:
nice